images/nbserve/nginx.py

```diff
@@ -76,47 +76,47 @@
     lua_ssl_verify_depth 10;
 
     # Serve things with appropriate mimetypes
-    include /etc/nginx/mime.types;
+    include /usr/local/openresty/nginx/conf/mime.types;
 
     # This is the 'regular' server, that sees all public
     # traffic and proxies them to the appropriate backend server.
     server {
         listen 0.0.0.0:8000;
 
-        location ~ \/\. {
-            deny all;
-        }
+        # location ~ \/\. {
+        #     deny all;
+        # }
```

Comment:
No, I suspect that's actually an artifact from experimentation. There shouldn't be any dotfiles really because people aren't ssh-ing in, but that doesn't mean we should assume there actually aren't any!  I'll add that to a commit here.

Reply:
Built the container with it and am deploying it now to make sure I didn't mess it up.

Reply:
Done!